FAQ.md

```diff
@@ -38,37 +38,34 @@ languages.
 
 # Licensing
 
-Slint is available under two licenses:
+Slint can be used under either a [commercial license](./LICENSES/LicenseRef-Slint-commercial.md) or [GNU GPLv3](./LICENSES/GPL-3.0-only.txt), at your choice.
 
- * GPLv3, for the growing ecosystem of Free and Open Source Software.
- * Commercial, for use in closed-source projects. See <https://slint-ui.com/#offering>
-
-The commercial license is free if you help us promote Slint: Check out our
+The commercial license can be provided for free if you help us promote Slint: Check out our
 [ambassador license](https://slint-ui.com/ambassador-program.html).
 
-## What is the commercial licensing model?
+## What are the commercial licensing options?
 
-The basic principle behind our commercial licensing is that you start for free and pay when you're shipping.
+We offer 3 options under the commercial license - Ambassador, Flex and Buyout. All 3 options include a non-exclusive, perpetual, irrevocable, non-transferable right to use Slint.
 
-We offer a perpetual license option and we generally deploy a per-product license, regardless of how many developers, designers, Q&A engineers are using Slint.
+The Ambassador license is a free license that can be provided in lieu of helping us promote Slint.
```

Comment:
In terms of terminology we talk about one commercial license (3 options under the license). Here we say that there's am "Ambassador license" and later a "Flex license". Would it make sense to use a different term? Like "Ambassador option", "Flex option" and "Buyout option"?